Example/Tests/StringAdditionsTests.swift

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright 2017 NextFaze
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy
+// of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+//
+//  StringAdditionsTests.swift
+//  FazeKit_Tests
+//
+//  Created by Ricardo Santos on 19/12/17.
+//
+
+import Foundation
+import XCTest
+import FazeKit
+
+class StringAdditionsTests: XCTestCase {
+    
+    override func setUp() {
+        super.setUp()
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+    }
+    
+    func testContains() {
+        let string = "Abcdefgh"
+        XCTAssertTrue(string.contains("A"), "String.contains: should have found a match")
+        XCTAssertFalse(string.contains("a"), "String.contains: should not have found a match")
+        XCTAssertFalse(string.contains("z"), "String.contains: should not have found a match")
+    }
+    
+    func testReplace() {
+        var string = "The cat jumped over the hat"
+        string = string.replace("cat", withString: "dog")
+        XCTAssertTrue(string.contains("dog"), "String.replace: should contain the replacement substring")
+        XCTAssertFalse(string.contains("cat"), "String.replace: should not still contain the original substring")
+    }
+    
+    func testSubscript() {
+        let string = "OMG. I can't believe it's a thing!"
+        XCTAssertTrue(string[0] == "O", "String.subscript: should return the Character O")
+        XCTAssertTrue(string[1] == "M", "String.subscript: should return the Character O")
```

Comment:
Assertion messages don't match 'M' and 'G'.

Reply:
Fixed 👍